lepton/sdk_example/create-ray-job.py

```diff
@@ -0,0 +1,268 @@
+                id=name,
+                name=name,
+                visibility=LeptonVisibility.PRIVATE if is_private else LeptonVisibility.PUBLIC
+            ),
+            spec=spec,
+        )
+
+    created = client.raycluster.create(lepton_raycluster)
+    if created:
+        print(f"Ray cluster {name} created")
+        print(f"Ray dashboard URL will be {client.url}/rayclusters/{name}/dashboard")
+    else:
+        print(f"Ray cluster {name} creation failed")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Create a Ray cluster with configurable node groups and environment variables",
+    )
+    parser.add_argument(
+        "--name",
+        type=str,
+        required=True,
+        help="Name of the Ray cluster"
+    )
+    
+    parser.add_argument(
+        "--node-group-name",
+        type=str,
+        required=True,
+        help="Name of the dedicated node group"
+    )
+    
+    parser.add_argument(
+        "--head-resource-shape",
+        type=str,
+        required=True,
+        help="Resource shape for head node (e.g., 'gpu.8xh200')"
+    )
+    
+    parser.add_argument(
+        "--worker-resource-shape",
+        type=str,
+        required=True,
+        help="Resource shape for workers (e.g., 'gpu.8xh200')"
+    )
+    
+    parser.add_argument(
+        "--image",
+        type=str,
+        default="ray:2.46.0",
+        help="Container image to use (default: ray:2.46.0)"
+    )
+    
+    parser.add_argument(
+        "--image-version",
+        type=str,
+        default="2.46.0",
+        help="Ray version (default: 2.46.0)"
+    )
+    
+    parser.add_argument(
+        "--image-pull-secret",
+        type=str,
+        default="",
+        help="Image pull secret name (default: empty string)"
+    )
+
+    parser.add_argument(
+        "--workers",
+        type=int,
+        default=1,
+        help="Number of worker replicas (default: 1)"
+    )
+    
+    parser.add_argument(
+        "--env",
+        action="append",
+        dest="env_vars",
+        default=[],
+        metavar="KEY=VALUE",
+        help="Environment variable in ENV_NAME=ENV_VALUE format. Can be specified multiple times."
+    )
+    
+    parser.add_argument(
+        "--secret",
+        action="append",
+        dest="secret_vars",
+        default=[],
+        metavar="ENV_NAME=SECRET_REF",
+        help="Secret environment variable in ENV_NAME=SECRET_REF format (e.g., WANDB_API_KEY=WANDB_API_KEY.zozhang). Can be specified multiple times."
+    )
+
+    parser.add_argument(
+        "--mount",
+        action="append",
+        dest="mounts",
+        default=[],        
+        metavar="PATH:MOUNT_PATH:STORAGE_TYPE:STORAGE_NAME",
+        help="Mount storage in PATH:MOUNT_PATH:STORAGE_TYPE:STORAGE_NAME format (e.g., /data:/workspace/data:pfs:my-storage). Can be specified multiple times."
+    )
+    
+    parser.add_argument(
+        "--is_private",
```

Comment:
nit-pick - should this be `--is-private` to match the dashes as seperators of the other flags?